custom_addons/non_conformance_report/views/x_ncr_nc_views.xml

```diff
@@ -45,22 +45,24 @@
                             <field name="uom"
                                    attrs="{'readonly': ['|', ('state', '=', 'ncr_submitted'), ('state', '=', 'received_vendor_response')]}"/>
                             <field name="cause_of_nc_id"
-                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': True}" options="{'no_create': True, 'no_create_edit':True}"/>
+                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': [('state', '=', 'ncr_submitted')]}"
+                                   options="{'no_create': True, 'no_create_edit':True}"/>
                             <field name="proposed_due_date"
-                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': True}"/>
+                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': [('state', '=', 'ncr_submitted')]}"/>
                             <field name="ca_response_id"
-                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': True}" options="{'no_create': True, 'no_create_edit':True}"/>
+                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': [('state', '=', 'ncr_submitted')]}"
+                                   options="{'no_create': True, 'no_create_edit':True}"/>
                         </group>
                         <group>
                             <field name="source_of_nc"
                                    attrs="{'readonly': ['|', ('state', '=', 'ncr_submitted'), ('state', '=', 'received_vendor_response')]}"
                                    options="{'no_create': True, 'no_create_edit':True}"/>
                             <field name="immediate_action"
-                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': True}"/>
+                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': [('state', '=', 'ncr_submitted')]}"/>
                             <field name="quantity" class="oe_inline"
                                    attrs="{'readonly': ['|', ('state', '=', 'ncr_submitted'), ('state', '=', 'received_vendor_response')]}"/>
                             <field name="disposition_type_id"
-                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': True}"
+                                   attrs="{'readonly': [('state', '=', 'received_vendor_response')], 'required': [('state', '=', 'ncr_submitted')]}"
```

Comment:
'readonly': [('state', '=', 'received_vendor_response') this means it will be editable in closed state . you should have it editable in part 2 only when the state is ncr_submitted